.github/workflows/e2e-tests.yml

```
@@ -4,6 +4,9 @@ on:
     branches:
       - master
   pull_request:
+    paths:
+      - '**'
+      - '!doc/**'
```

Nit: also ignore markdown files? 
```suggestion
      - '!doc/**'
      - '!**/*.md'
```

---

Yeah that's not a bad idea - any scenario where we would only want to restrict that to _root_ markdown files? Or extend this to changes made to similar files, like `.gitignore`? I'm mainly following suit w.r.t to the other workflows that disable that doc/ directory path, so maybe that's worth tackling down the line.